demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -611,6 +611,54 @@ def _is_approved_tags(self) -> bool:
                 return False
         return True
 
+    @error_codes('PA133')
+    def _is_approved_tag_prefixes(self) -> bool:
+        """Checks whether the tags in the pack metadata are approved
+
+        Return:
+            bool: True if the tags are approved, otherwise False
+        """
+        if tools.is_external_repository():
+            return True
+
+        is_valid = True
+        approved_prefixes = {x.value for x in list(MarketplaceVersions)}
+        pack_meta_file_content = self._read_metadata_content()
+        for tag in pack_meta_file_content.get('tags', []):
+            if ':' in tag:
+                tag_data = tag.split(':')
+                marketplaces = tag_data[0].split(',')
+                for marketplace in marketplaces:
+                    if marketplace not in approved_prefixes:
+                        if self._add_error(Errors.pack_metadata_non_approved_tag_prefix(tag, approved_prefixes), self.pack_meta_file):
+                            is_valid = False
+
+        return is_valid
+
+    def filter_by_marketplace(self):
+        """Filtering pack_metadata tags by marketplace"""
+        pack_meta_file_content = self._read_metadata_content()
+        common_tags, xsoar_tags, marketplacev2_tags = [], [], []
+        for tag in pack_meta_file_content.get('tags', []):
+            if ':' in tag:
+                tag_data = tag.split(':')
+                if 'xsoar' in tag_data[0]:
+                    xsoar_tags.append(tag_data[1])
+                if 'marketplacev2' in tag_data[0]:
+                    marketplacev2_tags.append(tag_data[1])
```

Comment:
```suggestion
                if 'xsoar' == tag_data[0]:
                    xsoar_tags.append(tag_data[1])
                elif 'marketplacev2' == tag_data[0]:
                    marketplacev2_tags.append(tag_data[1])
```

1. what if there's an invalid marketplace? we need an `else` case here
2. @daryakoval please make sure to handle this in your SDK-side PR

Reply:
1. There is a validation that checks the validity of the tag prefix

Reply:
great, please mention it in the docstring/comment, and commit the change to replace `in` with `==`

Reply:
I prefer using "in" as there may be multiple marketplaces in the tag prefix

Reply:
ah awesome, please document it so it's clearer to future readers too :)
is there a unit test checking a case of multiple marketplaces?

Reply:
@dorschw My SDK pr is already merged...

Reply:
@MLainer1 please make sure xpanse is supported here

Reply:
Working on it